Configuration/StandardSequences/python/Harvesting_cff.py

```diff
@@ -13,7 +13,7 @@
 from Validation.RecoMET.METPostProcessor_cff import *
 
 
-dqmHarvesting = cms.Path(DQMOffline_SecondStep*DQMOffline_Certification)
+dqmHarvesting = cms.Path(DQMOffline_SecondStep*DQMOffline_Certification*HLTMonitoringClient)
```

Comment:
the HLTMonitoringClient should be included in the DQM sequences in
DQMOffline_SecondStep_cff.py

not in Configuration/StandardSequences